src/util/commandUtils.ts

```diff
@@ -0,0 +1,215 @@
+/**
+ * Copyright (c) 2024 IBM Corporation.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+import * as Path from "path";
+import * as vscode from "vscode";
+import { isWin,mvnCmd,gradleCmd } from "../liberty/devCommands";
```

Comment:
What is the need to import methods `isWin`, `mvnCmd` and `gradleCmd` in `commandUtils`. These methods are never used in `devCommands` and elsewhere in the project. So in my suggestion you can remove these methods from `devCommands` and keep it in `commandUtils`. This can reduce one unnecessary import and improve performance.

Reply:
Comment incorporated. Moved unused methods from devCommands to commandUtils and removed import.
Thanks.

Reply:
Okay. Thanks